src/stories/index.js

```diff
@@ -1,19 +1,11 @@
 import React from 'react';
-
 import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
-import { linkTo } from '@storybook/addon-links';
+import App from '../App';
+import Title from '../components/Title';
+import Header from '../components/Header';
 
-import { Button, Welcome } from '@storybook/react/demo';
+storiesOf('App', module).add('to Storybook', () => <App/>);
 
-storiesOf('Welcome', module).add('to Storybook', () => <Welcome showApp={linkTo('Button')} />);
+storiesOf('Title', module).add('text', () => <Title/>);
```

Comment:
More descriptive names for the examples than `text` might be better - if it has only one possible state, perhaps `.add('default', () => <Title />);` makes sense